wasm/value.go

```diff
@@ -19,6 +19,13 @@ const (
 	ValueTypeI64 ValueType = 0x7e
 	ValueTypeF32 ValueType = 0x7d
 	ValueTypeF64 ValueType = 0x7c
+	// valTypeRange allows us to cache common valueType calculations. To determine the array offset, subtract
+	// valTypeFloor from the value of interest.
+	valTypeRange = ValueTypeI32 - valTypeFloor + 1
+	// valTypeFloor is the lowest numeric value of a value type.
+	//
+	// Note: the floor value changes after WebAssembly 1.0 (MVP)
```

Comment:
ps only reason I added this is I looked it up before punting to a larger array before, as it changes to https://webassembly.github.io/spec/core/binary/types.html#reference-types

I don't care about it, just made a comment to remind later.